apps/sports-helsinki/src/hooks/useRHHCConfig.tsx

```diff
@@ -58,6 +66,10 @@ export default function useRHHCConfig() {
         ArticleCardContent: (props: ArticleDetailsProps) => (
           <ArticleDetails {...props} />
         ),
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        LocationCardContent: (props: any) => (
```

Comment:
VenueCardContent would be a better name. The term "Venue" is used everywhere except in the CMS.

Reply:
I can change this. But. We will still have inconsistency, because in headless cms type is LocationSelected, so Location naming is obvious there.. But when we start to merge with venues api its Venue types, which is basically the same.. So in HCRC also a bit of a mess.